data/transactions/logic/assembler.go

```diff
@@ -1187,15 +1184,22 @@ func (ops *OpStream) Bytes() (program []byte, err error) {
 }
 
 func (ops *OpStream) addLine(err error) error {
```

Comment:
seems like this function is not used anymore, please remove.

Reply:
Thanks, removing.